app/models/grantee.rb

```diff
@@ -1,5 +1,5 @@
 class Grantee
-  attr_reader :id, :attributes
+  attr_reader :id, :attributes, :links
```

Comment:
[suggestion] I think you can remove this attr_reader, as I don't see anywhere it's being used now.

Reply:
it's in use [https://github.com/OHS-Hosting-Infrastructure/complaint-tracker/pull/190/files#diff-fde524589473ecc968bc7acfb135c1052ec8965b3c0b57c4c2fef5142f814ab8R16](here)